src/si/diffusion_coefficient.rs

```diff
@@ -0,0 +1,67 @@
+//! Diffusion coefficient (base unit square meter per second, m² · s⁻¹).
+
+quantity! {
+    /// Diffusion coefficient (base unit square meter per second, m² · s⁻¹).
+    quantity: DiffusionCoefficient; "diffusion coefficient";
+    /// Dimension of diffusion coefficient, L²T⁻¹ (base unit square meter per second, m² · s⁻¹).
+    dimension: ISQ<
+        P2,     // length
+        Z0,     // mass
+        N1,     // time
+        Z0,     // electric current
+        Z0,     // thermodynamic temperature
+        Z0,     // amount of substance
+        Z0>;    // luminous intensity
+    units {
+        @square_meter_per_second: prefix!(none); "m² · s⁻¹",
+            "square meter per second", "square meters per second";
+        @square_centimeter_per_second:  prefix!(centi) * prefix!(centi); "cm² · s⁻¹",
+            "square centimeter per second", "square centimeters per second";    
+        @square_millimeter_per_second:  prefix!(milli) * prefix!(milli); "mm² · s⁻¹",
+            "square millimeter per second", "square millimeters per second";                
+        @square_micrometer_per_second:  prefix!(micro) * prefix!(micro); "µm² · s⁻¹",
+            "square micrometer per second", "square micrometers per second"; 
+        @square_nanometer_per_second:  prefix!(nano) * prefix!(nano); "nm² · s⁻¹",
+            "square nanometer per second", "square nanometers per second";  
+        @stokes:  prefix!(centi) * prefix!(centi); "St",
+            "stokes", "stokes";    
+        @centistokes:  prefix!(centi) * prefix!(centi) * prefix!(centi); "cSt",
+            "stokes", "stokes";                                     
```

Comment:
* Changed to `m²/s` instead of `m² · s⁻¹`
 * Fixed some formatting.
 * `Stokes` is a person's name so I believe it should be upper case.
 * Fix `centistokes` description.

```suggestion
        @square_meter_per_second: prefix!(none); "m²/s", "square meter per second",
            "square meters per second";
        @square_centimeter_per_second: prefix!(centi) * prefix!(centi); "cm²/s",
            "square centimeter per second", "square centimeters per second";
        @square_millimeter_per_second: prefix!(milli) * prefix!(milli); "mm²/s",
            "square millimeter per second", "square millimeters per second";
        @square_micrometer_per_second: prefix!(micro) * prefix!(micro); "µm²/s",
            "square micrometer per second", "square micrometers per second";
        @square_nanometer_per_second: prefix!(nano) * prefix!(nano); "nm²/s",
            "square nanometer per second", "square nanometers per second";
        @stokes: prefix!(centi) * prefix!(centi); "St", "Stokes", "Stokes";
        @centistokes: prefix!(centi) * prefix!(centi) * prefix!(centi); "cSt", "centistokes",
            "centistokes";
```